README.md

```diff
@@ -57,15 +57,15 @@ Fork this repo, contribute and help keep it alive
 Small update to fix a security issue
 
 
-#### 2.8.5.4
+#### 2.8.5.4 - 3th of September 2021
```

Comment:
`3th` is not a thing.  `3rd` would be correct.

But personally I would do dates like: 2021-09-03 or 03-Sep-2021

Reply:
Agree, in fact I would argue for using 2021-09-03, as it's probably the most universal version

Reply:
Agreed, I will change it!